test/lib/mysql/index.spec.js

```diff
@@ -105,6 +105,108 @@ describe('lib/mysql', function() {
   })
 
 
+  describe('::findWhereEq', function() {
+
+    it('should execute a select query on the given table', function() {
+
+      const table = 'post_archive'
+
+      const request = {
+        projection : ['id', 'traversal_id', 'network_type_code']
+      , predicates : { stored_es : 1479439176, deleted : 0 }
+      , sort       : { id : 'desc' }
```

Comment:
If sort is an object this is bad.

How do we do multiple sorts (for instance last_name DESC, first_name DESC, id ASC)? 
Key ordering in objects is not guaranteed. We may need to use a tuple.